src/components/Dashboard/CompleteFiling.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="complete-ia-filing body-2">
+    <h4>Incorporation Complete</h4>
+
+    <p>{{name}} has been successfully incorporated.</p>
+    <p>Return to your Manage Businesses dashboard to access your business and file changes.</p>
+
+    <div class="to-dashboard-container">
+      <v-btn
+        color="primary"
+        @click.native.stop="returnToDashboard()"
+        >
```

Comment:
Think the end of this tag has an extra tab space

Reply:
fixed

Reply:
I see this is now fixed.